common/src/main/java/io/druid/metadata/MetadataStorageConnector.java

```diff
@@ -44,4 +44,6 @@ public Void insertOrUpdate(
   public void createConfigTable();
 
   public void createTaskTables();
+
+  public void createAuditTable();
```

Comment:
This is a high level question. Can you explain the decision to create a specific audit table versus adding a new column to existing schemas? For examples, with rules, we store all older variations of an edited rule. If we choose to have an audit table, we can also change the rule logic such that we edit rules versus creating new entries every time. I am cool with that